### DISCUSSION TITLE
question about input VCF

### DISCUSSION THREAD

**Discussion by XieHongX** [LohseLab/gIMble#130] [Q&A] [answered]
Hi I have two questions about input vcf:
1. Is it still only support vcf file from freebayes? I have vcf file from bcftools pileup.
2. When providing the raw vcf, should it be the raw file from freebayes or can I filter the vcf file before gimbleprep (for example filter using bcftools)?

**Comment by GertjanBisschop** [comment, marked as answer]
1. I think the answer is yes here.
2. I think filtering should be fine as long as the resulting file still complies with what gimble expects as input. You could probably test this yourself and check whether any of the gimble dependencies raises an issue.

**Reply by XieHongX** [reply]
Hi - I did a test. gimbleprep now runs well with vcf from other sources and pre filtering is OK. But I am yet to explore if that will make  some difference in the final results
